packages/node/src/sdk.ts

```diff
@@ -149,7 +156,20 @@ export async function close(timeout?: number): Promise<boolean> {
 }
 
 /**
- * A function that returns a Sentry release string dynamically from env variables
+ * Function that checks if autoSessionTracking option is enabled
```

Comment:
What happens if multiple clients are used? This function and then all the code that relies on it only looks at the current hub / current client.

We don't have a good story for multiple clients on the browser, I'm not sure about the current state in Node, but this just makes matters worse.

Reply:
You are right! in that case it should probably accept the client as an argument